### PR TITLE
4184 schema org bug

### PR DESCRIPTION
**Pull request by hyi**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Comment by hyi** [comment]
@sblack-usu Can you take a quick look of this small PR that will fix the schema.org bug in issue #4184. I have used the validation tool and validated the fix on beta, so it should just work and the review should be straightforward as well.

File: hs_core/templates/pages/genericresource.html [hunk 1]
```diff
@@ -99,10 +99,10 @@
 	"license": {
 	    "@type": "CreativeWork",
 	    {% if cm.metadata.rights.statement and cm.metadata.rights.url %}
-        "text": "{{ cm.metadata.rights.statement }}",
+        "text": "{{ cm.metadata.rights.statement | escapejs }}",
```
Comment (sblack-usu) [inline hs_core/templates/pages/genericresource.html:102]: Are there any other fields that should be escaped?

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|
| NEW SECURITY HOTSPOTS | 0 |
| NEW DUPLICATED LINES DENSITY | 0.0 |
| NEW CODE SMELLS | 0 |
| NEW VULNERABILITIES | 0 |
| NEW BUGS | 0 |

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4194](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4194)

File: hs_core/templates/pages/genericresource.html (hunk 1 above)
Comment (sblack-usu) [reply hs_core/templates/pages/genericresource.html:102]: https://github.com/hydroshare/hydroshare/issues/4184#issuecomment-811473710

Comment (hyi) [reply hs_core/templates/pages/genericresource.html:102]: Good question. Abstracts/description field was already escaped in the initial implementation. I checked all other fields and verified this license statement field is the only other field that could have this issue.

**Comment by hyi** [comment]
@sblack-usu Have escaped a couple of more fields just to be on the safe side. See if it looks good now.

**Review by sblack-usu — APPROVED** [review]

**Pull request merged**